doc/tools/west/repo-tool.rst

```diff
@@ -327,5 +327,47 @@ These commands perform miscellaneous functions.
   that fetches upstream data is run (this behavior can be
   suppressed for the duration of a single command by passing ``--no-update``).
 
+.. _west-mr-build:
+
+Build system integration
+************************
+
+Some of the projects managed by west require to be integrated with the Zephyr
+build system, which is based around Kconfig and CMake (see
+:ref:`application` for more information about Zephyr's build system).
```

Comment:
Suggest adding:

Examples include vendor HALs and other firmware support libraries.